modular_skyrat/modules/modular_reagents/code/reagents/medicine.dm

```diff
@@ -25,6 +25,84 @@
 	M.adjustOrganLoss(ORGAN_SLOT_HEART,3 * REM * delta_time, 80)
 	..()
 
+
+/datum/reagent/medicine/ataraxydone
+	name = "Ataraxydone"
+	description = "A stasis-inducing chemical used for exotic surgeries or emergency medicine."
+	taste_description = "liquid"
+	reagent_state = LIQUID
+	color = "#e0d394" // 224, 211, 148
+	metabolization_rate = 0.2 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+	ph = 7
+	purity = REAGENT_STANDARD_PURITY
+	creation_purity = REAGENT_STANDARD_PURITY
+	impure_chem = /datum/reagent/impurity/ataraxydone
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED | REAGENT_DEAD_PROCESS | REAGENT_IGNORE_STASIS
```

Comment:
It will process in dead people? Will that do like formaldehyde, or will it actually be metabolized while dead?